images/base/files/usr/local/bin/entrypoint

```diff
@@ -416,10 +437,18 @@ enable_network_magic(){
         echo "ERROR: Have an old IPv4 address but no current IPv4 address (!)" >&2
         exit 1
       fi
-      # kubernetes manifests are only present on control-plane nodes
-      sed -i "s#${old_ipv4}#${curr_ipv4}#" /etc/kubernetes/manifests/*.yaml || true
-      # this is no longer required with autodiscovery
-      sed -i "s#${old_ipv4}#${curr_ipv4}#" /var/lib/kubelet/kubeadm-flags.env || true
+      if [[ "${old_ipv4}" != "${curr_ipv4}" ]]; then
+        # kubernetes manifests are only present on control-plane nodes
+        sed_ipv4_command="s#$(regex_escape_ip "${old_ipv4}")#${curr_ipv4}#g"
```

Comment:
note that when I did the regex fix I also:
- stopped computing this string repeatedly
- changed it to a global replace (!) instead of just replacing the first instance

the existing sed pattern carried forward from the existing script was not good :(
it should be reasonable now.

Reply:
Thanks!
Reminded by it, I also added word boundary to sed pattern to avoid mismatch like
```
# sed "s/1\.1\.1\.2/1.1.1.3/g" <<< "1.1.1.22"
1.1.1.32
```